### PR TITLE
Add a custom transpiler binary for watching and transpiling lib files

### PR DESCRIPTION
**Pull request by jbwyme**

### CODE REVIEW

File: transpile.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+          'extensions': ['.svg'],
+        }],
+      ],
+      presets: ['es2015'],
+    };
+    return  babel.transformFileSync(file, options).code;
+  },
+
+  jade: file => {
+    const source  = fs.readFileSync(file).toString();
```
Comment (tdumitrescu) [inline transpile.js]: twooooooooo space!

File: transpile.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    const vjade = require('virtual-jade');
+    const vjadeOptions = {
+      filename: file,
```
Comment (tdumitrescu) [inline transpile.js]: cool, just to make sure, have you tested this with jade templates including other files?

File: transpile.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+      .render()
+      .replace(/\n/g, ' ')
+      .replace(/"/g, /\/"/);
+    return 'module.exports = "' + css + '";\n';
+  },
+};
+
+function transpileFile(file) {
+  const ext = file.split('.').pop();
+  if (['js', 'jade', 'styl'].indexOf(ext) > -1) {
```
Comment (tdumitrescu) [inline transpile.js]: how about
```js
if (['js', 'jade', 'styl'].includes(ext)) {
```

File: transpile.js (hunk 2 above)
Comment (jbwyme) [inline transpile.js]: yeah we have templates that inline svgs in the source already

**Pull request merged**